README.md

```diff
@@ -101,19 +101,19 @@ describe('<MyComponent />', () => {
   });
 
   it('renders children when passed in', () => {
-    const wrapper = shallow((
+    const wrapper = shallow(
```

Comment:
In fact, this is intentional and required, or else the linter requires a trailing comma after `</MyComponent>`